src/components/getting-started/fileblock.js

```diff
@@ -23,110 +23,87 @@ import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 
 var Modernizr = window.Modernizr;
 
-class FileBlock extends React.Component {
-  state = {
-    hovering: false,
-  };
+export const FileBlock = ({ children, hideText, fileName }) => {
+  const [isHovering, setHovering] = useState(false);
+  const [isCopiedToClipboard, setCopyToClipboard] = useCopyToClipboard();
 
-  copyCodeToClipboard = e => {
-    e.preventDefault();
+  const preElement = useRef(null);
 
-    copy(Helpers.dedent(this.props.children));
+  const copyCodeToClipboard = e => {
+    e.preventDefault();
 
-    this.setState({ clicked: false });
+    const contentToCopy = Helpers.dedent(children);
```

Comment:
It would be nice and a good practice to import just the function we are using from `Helpers`. 
I know you found it like this, but since we are refactoring this now, we can quickly improve change this as well.